Bugfix:  disable opacity slider for opaque blending (#5088)
Napari/napari: docs/guides/layers.md

`opaque`. These modes determine how the visuals for this layer get mixed with
the visuals from the other layers.

> * An `opaque` layer renders all the other layers below it invisible.

Reviewer: ```suggestion
* An `opaque` layer hides any layer data below it.
```

I feel like saying "renders invisible" might be a bit confusing, especially since layers have a `visibile` attribute. They are still visible where the layer does not have any data, they are just hidden where there *is* data.
